app/search_builders/curation_concerns/filter_by_type.rb

```diff
@@ -3,7 +3,7 @@ module FilterByType
     extend ActiveSupport::Concern
 
     included do
-      self.default_processor_chain += [:filter_models]
+      self.default_processor_chain += [:filter_models, :only_active_works]
```

Comment:
I don't think this belongs in `FilterByType`  Instead can you make a  new module called `FilterByStatus`?